includes/class-wc-payments-order-service.php

```diff
@@ -1270,16 +1270,16 @@ private function generate_dispute_closed_note( $dispute_id, $status ) {
 	/**
 	 * Composes url for dispute details page.
 	 *
-	 * @param string $dispute_id Dispute id.
+	 * @param string $charge_id Charge ID.
```

Comment:
To be clear and consistent with what you have in other function doc-block.

```suggestion
	 * @param string $charge_id The disputed charge ID.
```